src/misc/programs.cpp

```diff
@@ -121,7 +140,7 @@ Program::Program() {
 	psp = new DOS_PSP(dos.psp());
 	/* Scan environment for filename */
 	PhysPt envscan=PhysMake(psp->GetEnvironment(),0);
-	while (mem_readb(envscan)) envscan+=mem_strlen(envscan)+1;	
+	while (mem_readb(envscan)) envscan+=mem_strlen(envscan)+1;
```

Comment:
Please use clang format; spaces should be added around operators, etc.

Reply:
If you're on a Bourne/SH capable system:

run: `./scripts/format-commit.sh`

And it will carefully format just the current commit's (and pending) lines (it might also cleanup function signatures).